server/graphql/v2/object/Individual.js

```diff
@@ -107,6 +108,12 @@ export const Individual = new GraphQLObjectType({
           }
         },
       },
+      hasSeenLatestChangelogEntry: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        async resolve(individual) {
+          return await hasSeenLatestChangelogEntry(individual);
```

Comment:
Did you try this one to confirm if it works? My understanding is that `individual` is a `Collective`, not a `User`

Reply:
Good feedback. Do we have any test coverage on this?

Reply:
@Betree : Ops, you are right, seems I've missed this, since we aren't using this field for now. Anyways, I've tested it and added a line to convert it to user first before passing. 👍🏽